packages/@jsii/go-runtime/jsii-experimental/runtime.go

```diff
@@ -36,6 +36,26 @@ func Load(name string, version string, tarball []byte) {
 	}
 }
 
+// Create will construct a new JSII object within the kernel runtime. This is
+// called by jsii object constructors.
+func Create(fqn string, args []interface{}, interfaces []string, overrides []Override, returns interface{}) {
```

Comment:
I wonder if we should alias `string` to `FQN` in order to make the type name parameters more expressive?